PublicationsRouter/api/v3/Retrieve.md

```diff
@@ -129,17 +126,15 @@ In order to tell the difference between (1) and (2), compare the following two l
             "packaging" : "https://pubrouter.jisc.ac.uk/FilesAndJATS"
         },
         {
-            "type" : "fulltext",
-            "format" : "application/pdf",
-            "url" : "https://pubrouter.jisc.ac.uk/api/v3/notification/123456789/content/publisher.pdf",
+            "type": "fulltext", 
+            "format": "application/pdf", 
+            "url": "https://externallink.ac.uk/123456789"
         }
     ]
+    
+The first link is a packaged binary link, and includes the packaging type used to package the binary. Packaging types are explained in further detail in the next section. 
 
-The first link has type "package" and also has an element **packaging** which tells you this is of the format "https://pubrouter.jisc.ac.uk/FilesAndJATS".
-
-The second link does not contain a **packaging** element at all, and does not have "package" as its type.
-
-This means the first link is a link to package held by PubRouter, and the second is a proxy for a URL hosted by the publisher.
+The second link is to an external site, this will not have a packaging type. The type field will be describing the type of content (likely fulltext), and the format field describes the content's format (likely application/pdf, or text/html). 
```

Comment:
I suggest here that fulltext is either italicized or put in quotes, ditto each of application/pdf and text/html.